docs/rst/user_manual/configuration.rst

```diff
@@ -65,9 +119,36 @@ with type name ``HelloWorld``.
       - name: HelloWorldTopic
         type: HelloWorld
 
-.. note::
+    blocklist:
+      - name: "*"
+        type: HelloWorld
```

Comment:
Use real cases to exemplify. rq and rr are the name for topics in services in ROS2 
```suggestion
      - name: "rq/*"
      - name: "rr/*"
```

Reply:
The idea was to filter them, so the whole .yaml refers to a real use case.